src/styles/Books.css

```diff
@@ -0,0 +1,23 @@
+.contain {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0 35px 40px;
+}
+
+.content {
+  width: 90%;
+  max-width: 1820px;
+  margin-top: 50px;
+  background-color: #fafafa;
+}
+
+.header {
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 95%;
+}
```

Comment:
- [ ] Please fix your font and button style it doesn't match that of the [Zeplin](https://zeplin.io/) design as requested by the previous reviewer.
**Zeplin Design**
![zeplin](https://user-images.githubusercontent.com/55596970/224482218-99d39429-d865-48d3-bb4f-3c9b1c996f1c.png)
**Your Design**
![fix](https://user-images.githubusercontent.com/55596970/224482227-63eaa244-0e3c-4cf5-8c8f-51975756181a.png)

Reply:
hey @CollinsTatang  thanks for the review, but I think we have different screen resolutions, this is a video showing my project on my screen. https://drive.google.com/file/d/1Aa9D4JkZMLP6yucTH8LKZhVMFaDMv5Mk/view?usp=sharing

Reply:
Ok @alicemirigo92 
I see from the video you are right 
Please correct the remove button to match the design